src/main/java/nl/thehyve/ocdu/services/ValidationService.java

```diff
@@ -91,6 +93,10 @@
         PatientDataOcChecks checksRunner = new PatientDataOcChecks(metadata, bySubmission);
         errors.addAll(checksRunner.getErrors());
 
+        if(errors.size() == 0) {
+            openClinicaService.registerPatients(submitter.getUsername(), wsPwdHash, submitter.getOcEnvironment(), bySubmission);
```

Comment:
ValidationService is meant only for Validation - not for registering patients. 
The control flow in Controller responsible for handling patient view should be as follows:
- validate (using validation service)
- if validation yielded no errors: call OpenClinicaService.registerPatients